tests/test_ingest.py

```diff
@@ -43,6 +43,7 @@ class MegaPrimeIngestTestCase(IngestTestBase, lsst.utils.tests.TestCase):
     ingestDir = os.path.dirname(__file__)
     instrumentClassName = "lsst.obs.cfht.MegaPrime"
     rawIngestTask = "lsst.obs.cfht.MegaPrimeRawIngestTask"
+    filterLabel = lsst.afw.image.FilterLabel(physical="i.MP9702", band="i2")
```

Comment:
I realize this is the existing convention for CFHT, but it will make code that checks "r2" and "i2" even more risky...

Reply:
I'm just going off of the definitions in `cfhtFilters.py`.